dockerfiles/base/scripts/base/library.sh

```diff
@@ -345,9 +344,17 @@ verify_version_compatibility() {
   ##      - If they don't match and one is nightly, fail
   ##      - If they don't match, then if CLI is older fail with message to get proper CLI
   ##      - If they don't match, then if CLLI is newer fail with message to run upgrade first
-
   CHE_IMAGE_VERSION=$(get_image_version)
 
+  # Only check for newer versions if not in offline mode.
+  if ! is_offline; then
+    NEWER=$(compare_versions $CHE_IMAGE_VERSION)
```

Comment:
If I use a tagged version ( not latest alias) I'm not sure we shouldn't do the check. Only in case of latest